indico/modules/events/registration/placeholders/registrations.py

```diff
@@ -31,6 +31,20 @@ def render(cls, regform, registration):
         return registration.last_name
 
 
+class PicturePlaceholder(Placeholder):
+    name = 'picture'
+    description = _('Picture of the person')
+
+    @classmethod
+    def render(cls, regform, registration):
+        picture_data = registration.get_personal_data_picture()
+        if not picture_data:
+            return _('NO PICTURE')
```

Comment:
That looks quite ugly, especially w/ the uppercase.

Two potential options:

- Returning an empty string
- Returning a placeholder image (like the one with the initial we use e.g. in meeting participant lists)